record-editor/src/app/shared/components/extract-actions/author-extract-actions.component.ts

```diff
@@ -99,7 +97,7 @@ export class AuthorExtractActionsComponent {
 
   private onError(error: ApiError) {
     this.toastrService.clear(this.extractingToast.toastId);
-    if (error.status === 500 && error.message) {
+    if (error.status === 400 && error.message) {
```

Comment:
do you need to do anything on `next` to take these changes into account (bump version or something like that)?

Reply:
I'll have to change it manually because authorlist functionality needs to stay at next to for the view that we are using (or we can leave status code 500).